.github/PULL_REQUEST_TEMPLATE/back-end-pr-template-systems-intelligence.md

```diff
@@ -45,12 +45,12 @@
     - [ ] AC2
     - [ ] ...
 - [ ] Pull Request is linked to Asana task
+- [ ] CR has been offered to any other teams who the changes might effect
```

Comment:
```suggestion
- [ ] CR has been offered to any other teams who the changes might affect
```
Typo.

Reply:
More importantly, what does "offer" mean in this context? I wonder if a simple "other teams who might be affected by the changes have been notified" would be better or even in addition to the CR.

Otherwise, I could see something akin to -- 
> PT-Sierra: We have this PR up for code review. Tango, anyone want to review this?
> PT-Tango: We're trying to get a release out, no one is available to review, Sierra.
> PT-Sierra: Okay, then we're pushing
> PT-Tango: Wait. Why did all our API calls break?

Which is obviously a contrived example, but my point is that we should be _proactive_. It's not enough to passively "offer", we need to actively "notify". Or both! But I think we can take a more active stance here.

Reply:
To my ear, someone who has been offered a chance to code review has been notified :shrug:

But, I definitely don't have any strong opinions about the word choice here as long as the speedbump to think about the impact on other teams is in place. In https://github.com/nicheinc/.github/pull/3/commits/ba7e99fd43a66855da1292d27b39d58b02a318f6 I reworded the bullet point to read: 
> - [ ] Other teams who might be affected by the changes have been notified and
  offered a chance to CR